core/constants.py

```diff
@@ -22,7 +22,7 @@ class PhaseState(Enum):
 
 
 class Signal(Enum):
-    START_FEEDING_PHASE = auto()
+    BEFORE_FEEDING_PHASE = auto()  # Note to B: See Green box on p.7 of rules regarding the play order. I also note that there will need to be an order in which these BEFORE_FEEDING_PHASE cards are activated. I suggest, 1. Fertile, 2. Fat Tissue, 3. Long Neck.
```

Comment:
Noted, maybe we should then explicitly bind these to phases of the game (where it makes sense to do so; I suspect we may well find cases that don't fit that model exactly, but perhaps we can do this for those that do). So we could call this PRE_MODIFY_ENVIRONMENT? Then if we want to have signals that happen at the end of a phase we could have e.g. POST_MODIFY_ENVIRONMENT? What do you reckon?

Reply:
Or maybe we leave this as a Signal i.e. we still have a START_FEEDING_PHASE (and we can also have an END_FEEDING_PHASE one too if necessary) but just bind the Long Neck stuff to a new event START_MODIFY_ENVIRONMENT_PHASE?